tests/vmi_multus_test.go

```diff
@@ -593,21 +579,41 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
-		// Create two sriov networks referring to the same resource name
+		// Extra cleaning of sriov networks
 		result := virtClient.RestClient().
+			Delete().
+			RequestURI(fmt.Sprintf(sriovNetworksEndpoint, "")).
+			Do()
+		Expect(result.Error()).NotTo(HaveOccurred())
+
+		// Create two sriov networks referring to the same resource name
+		result = virtClient.RestClient().
 			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov")).
+			RequestURI(fmt.Sprintf(sriovNetworksEndpoint, "sriov")).
 			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov", tests.NamespaceTestDefault))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 		result = virtClient.RestClient().
 			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov2")).
+			RequestURI(fmt.Sprintf(sriovNetworksEndpoint, "sriov2")).
 			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov2", tests.NamespaceTestDefault))).
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
+	AfterEach(func() {
```

Comment:
Could you mention and explain this in the commit message please?

Reply:
Changed